ITR/data/template.py

```diff
@@ -173,7 +172,7 @@ def _fixup_name(x):
         if template_version==2:
             esg_data_sheet = TabsConfig.TEMPLATE_ESG_DATA_V2
             try:
-                df_esg = df_company_data[esg_data_sheet].drop(columns='company_lei').copy().iloc[0:45]
+                df_esg = df_company_data[esg_data_sheet].drop(columns='company_lei').copy() # .iloc[0:45]
```

Comment:
what was the reason to have `.iloc[0:45]`, and why don't we need it any more?

Reply:
When I was debugging the code it was helpful to just look at the first two companies only, not all 70.  It so happened those first two companies were the first 45 rows of data...We can remove.